### DISCUSSION TITLE
Amazingly efficient resource usage - Golang PocketBase

### DISCUSSION THREAD

**Discussion by MengLinMaker** [pocketbase/pocketbase#4263] [Show and tell]
This is just a simple benchmark not indicative of real world performance.

The test is a simple query to return the id of an admin using PocketBase as a Golang framework. The backend is deployed to fly.io via a docker, configured with:

- 256 MB RAM
- 1 vCPU
- Less than 100 MB image size with Litestream streaming replication.

This is probably more of a stress test for the Golang echo framework used by PocketBase. Even so, the bottleneck is probably caused by the amount of requests I can make from my laptop.

Average request per second sent from laptop is 326.51

<img width="820" alt="image" src="https://github.com/pocketbase/pocketbase/assets/39476147/3446ca2a-6e43-45cb-844d-f9c45ca47b19">

**Comment by ganigeorgiev** [comment]
_Side-note: If you are looking for benchmarks with various dataset sizes and loads, you can also check the results from https://github.com/pocketbase/benchmarks._
